xdsl/transforms/experimental/stencil_tensorize_z_dimension.py

```diff
@@ -262,29 +267,18 @@ def is_scalar(typ: Attribute) -> TypeGuard[AnyFloat]:
     return isinstance(typ, AnyFloat)
 
 
-class ExternalLoadOpTensorize(RewritePattern):
-    @op_type_rewrite_pattern
-    def match_and_rewrite(self, op: ExternalLoadOp, rewriter: PatternRewriter, /):
-        if not is_tensorized(op.result.type):
-            assert isa(op.result.type, FieldType[Attribute])
-            rewriter.replace_matched_op(
-                ExternalLoadOp.get(op.field, stencil_field_to_tensor(op.result.type))
-            )
-
-
```

Comment:
Is this no longer needed

Reply:
It's handled by the type conversion pass.